src/install/dracut-install.c

```diff
@@ -1839,8 +1837,8 @@ static int modalias_list(struct kmod_ctx *ctx)
                                 continue;
                         kmod_list_foreach(l, modlist) {
                                 mod = kmod_module_get_module(l);
-                                char *name = strdup(kmod_module_get_name(mod));
-                                hashmap_put(modules_loaded, name, name);
+                                char *modname = strdup(kmod_module_get_name(mod));
```

Comment:
This can be just `name = strdup(kmod_module_get_name(mod));` , no need to declare a new variable.